general/networking/Fail2ban.md

```diff
@@ -0,0 +1,77 @@
+---
+uid: Fail2ban-Jellyfin
+title: Fail2ban
+---
+
+## Fail2ban
+
+Fail2Ban is an intrusion prevention software framework that protects computer servers from brute-force attacks.
+Fail2Ban operates by monitoring log files (e.g. /var/log/auth.log, /var/log/apache/access.log, etc.) for selected entries and running scripts based on them.
+
+Jellyfin produces log that can be monitored by Fail2ban to prevent brute-force attacks on remote instance of jellyfin.
+
+### Requirements
+
+* Jellyfin remotely accessible
+* Fail2ban installed and running
+* Knowing where the log for Jellyfin are stored : by default /var/log/jellyfin/
+
+### Step One create a jail
```

Comment:
```suggestion
### Step One: Create a Jail
```

Reply:
The other steps should also use a colon.

Reply:
I'd also remove the capital letter from "One"